rskj-core/src/main/java/co/rsk/net/NodeMessageHandler.java

```diff
@@ -206,23 +201,25 @@ private boolean allowByMinerNotBanned(Peer sender, Message message) {
      * record event if message is repeated
      */
     private boolean allowByMessageUniqueness(Peer sender, Message message) {
```

Comment:
the logic of this method has change a bit.. could you please make it visible for tests and cover different inputs and expected results?

Reply:
Addressed